draft-irtf-cfrg-vdaf.md

```diff
@@ -5098,10 +5098,40 @@ possible. See {{xof-vs-ro}} for more information.
 | FieldLeaf  | `Field255` ({{fields}}) |
 {: #idpf-bbcggi21-param title="Constants and type definitions for the IDPF of BBCGGI21."}
 
-### Key Generation
+### Overview
+
+On a high level, the IDPF maps a key generator's input `(alpha, beta_1, ..., beta_BITS)` onto
+a binary tree with `BITS+1` levels, where each edge going from a parent node to a left child is
+labeled `0`, and each right edge is labeled `1`. Then each leaf node corresponds to a value
+`x` in `0, ..., 2^BITS`, and the labels on the path from the root to `x` contain the binary
+decomposition of `x`, starting with the most significant bit. Finally, all inner nodes have
+the label `0`, while the nodes on the path from the root to `alpha` have labels
+`beta_1, ..., beta_BITS`.
+
+The IDPF construction now boils down to secret-sharing the node labels of that tree in an efficient way. Note
+that explicitly representing the tree requires `O(2^BITS)` space, so the generator cannot just
+compute additive shares of it and send them to the two evaluators. Instead, the evaluators will
+re-generate shares of the tree using a pseudorandom generator (PRG). The basic observation is that if
+both evaluators have the same seed `s` of length `KEY_SIZE`, then Expanding `s` to length `2*KEY_SIZE`
+using a PRG, then the expansion can be split again into two seeds `s_l`, `s_r`, that can again serve
```

Comment:
I think this sentence needs to be rephrased

Reply:
Done.